akka-http/src/main/scala/akka/http/scaladsl/server/Route.scala

```diff
@@ -73,8 +73,10 @@ object Route {
                                 exceptionHandler: ExceptionHandler         = null): Flow[HttpRequest, HttpResponse, NotUsed] =
     Flow[HttpRequest].mapAsync(1)(asyncHandler(route))
 
-  def asyncHandler(route: Route, routingSettings: RoutingSettings, parserSettings: ParserSettings)(implicit system: ClassicActorSystemProvider): HttpRequest => Future[HttpResponse] = {
+  def toFunction(route: Route)(implicit system: ClassicActorSystemProvider): HttpRequest => Future[HttpResponse] = {
```

Comment:
Is this just about naming?

Reply:
No: akka-grpc was using the `asyncHandler` that takes a `Route` and a bunch of implicits, and had to replace it with this method that required explicit RoutingSettings and ParserSettings. Removing those, however, would clash with the existing asyncHandler.

Renaming seemed elegant (with `toFunction` being similar to `toFlow`).

Alternatively we could change the deprecated `asyncHandler` and turn some implicit parameters into explicit ones, but that does look a bit strange. I have no strong preference.

Reply:
Ah, I see, thanks for the explanation.

I find `toFunction` a bit unexpected as `Route` already is an alias for a function. But maybe people don't even care about that (and most people wouldn't use it anyway). So, fine for me!